go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/philoserf-contribs/kube-janitor
```

Comment:
@philoserf Should this be `module github.com/theMagicalKarp/kube-janitor` to match the repo. Thanks for making a contribution btw!